lib/jsonLogger.js

```diff
@@ -14,6 +14,9 @@ const getLogMethod = level => {
 const isError = obj => Object.prototype.toString.call(obj) === '[object Error]'
 
 const getMessage = obj => {
+  if (obj == null) {
+    return obj;
+  }
```

Comment:
A little change to fit with the project's linting.

```suggestion
  if (obj === null || obj === undefined) {
    return obj
  }
```